agogosml_cli/Makefile

```diff
@@ -102,4 +105,10 @@ installedit: clean ## install the package while dynamically picking up changes t
 	pip install --editable .
 
 dockerbuild: clean
-	docker build -t agogosml_cli .
+	@if [ ! -z "${TOX_VERSION}" ]; then \
+		echo "buidling all"; \
```

Comment:
Should this log be `echo "building ${TOX_VERSION}"` since `TOX_VERSION` may have a value other than `all`?